src/Menu/Menu.spec.tsx

```diff
@@ -0,0 +1,93 @@
+import "@testing-library/jest-dom/extend-expect";
+import * as faker from "faker";
+import React from "react";
+import userEvent from "@testing-library/user-event";
+import { Button } from "../Button";
+import { cleanup, render } from "@testing-library/react";
+import { Menu } from "../Menu";
+import { MenuItem } from "../MenuItem";
+import { SpaceKitProvider } from "../SpaceKitProvider";
+
+afterEach(cleanup);
+
+test("when child of `Menu` is clicked, menu is shown", () => {
+  const menuItemText = faker.random.word();
+  const triggerText = faker.random.word();
+
+  const { getByText, queryByText } = render(
+    <SpaceKitProvider disableAnimations>
+      <Menu
+        content={
+          <>
+            <MenuItem>{menuItemText}</MenuItem>
+          </>
+        }
+      >
+        <Button>{triggerText}</Button>
+      </Menu>
+    </SpaceKitProvider>
+  );
+
+  expect(queryByText(menuItemText)).not.toBeInTheDocument();
+  userEvent.click(getByText(triggerText));
+  getByText(menuItemText);
```

Comment:
`getByText` intrinsically is an assertion. `getBy*` methods throw if they don't synchronously find what you're looking for. Adding `expect(getByText()).toBeInTheDocument()`, while it's more what we're used to, is superfluous.

I had this conversation with @jgzuke too and asked him to trust me that this will become second nature. If we find that this burns us somehow in the future, then let's revisit it.

Reply:
gotcha, thx for the explanation.  it _seems_ like an inconsistency and not quite intuitive though.  will leave it up to you.

Reply:
@timbotnik You're totally right that it seems inconsistent and not intuitive. I'm weighing this against the other option:

```js
expect(getByText(menuItemText)).toBeInTheDocument()
```

If `getByText` fails, then the code with `throw` and the `expect` will never get checked (so the error message won't actually be something related to being in the document). If it returns, then the expect will _always_ pass. 

I prefer the concise version. If we find this causes us problems then we should talk about it :-D